src/SpecialFunctions.jl

```diff
@@ -71,7 +71,12 @@ end
 export sinint,
        cosint
 
+export ellipj,
+       jss,jsc,jsd,jsn,jcs,jcc,jcd,jcn,jds,jdc,jdd,jdn,jns,jnc,jnd,jnn,
```

Comment:
What are all these exports?

Reply:
Ah, I see, they're generated by macros. The names are a bit cryptic: are these standard? Maybe prefix them (`elliptic_jss`) or stick them in a module (`Elliptic.jss`)?

Reply:
There are 16 Jacobi elliptic functions, namely all pairs of the letters scdn. Since `cd` is already taken, I decided to put a `j` (for Jacobi) in front of the function to avoid name collision. Elliptic.jl solved this issue by putting these functions into a module `Jacobi`, but I'd say `jsn(u,m)` is more convenient than `Jacobi.sn(u,m)`. Might be worth discussing, though.

Reply:
ah ok, how about `jacobisn`? This matches nicely with `besselk`/`besselj`, etc.

Reply:
`jacobisn` possibly makes it hard to distinguish the functions, and it wouldn't really save much typing compared to `Jacobi.sn`. It might be worth to get input from someone who uses these functions regularly, though.

Reply:
Looking at other packages:
- Mathematica: `JacobiSN`
- Matlab & SciPy provides them all at once via `ellipj` function.
- GSL provides them all via `gsl_sf_elljac_e`

Reply:
I'd still prefer to rename the functions to longer names.